### PR TITLE
docs(zeebe): document configuration for GCS backup store

### PR DESCRIPTION
**Pull request by oleschoenburg**
## What is the purpose of the change

This documents a new backup store for Google Cloud Storage. We [already supported S3](https://docs.camunda.io/docs/next/self-managed/backup-restore/zeebe-backup-and-restore/#configure-s3-backup-store) but with 8.2, Zeebe ships with [native support for GCS](https://github.com/camunda/zeebe/issues/11917).

## Are there related marketing activities

I'm not sure what this means - probably not?

## When should this change go live?

With the release of Camunda Platform 8.2.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by deepthidevaki — APPROVED** [review]
Thanks. LGTM

File: docs/self-managed/backup-restore/zeebe-backup-and-restore.md [hunk 1]
```diff
@@ -70,20 +74,49 @@ You can enable compression by specifying a compression algorithm to use. We reco
...
+The bucket specified with `bucketName` **must already exist**, Zeebe will not try to create one for you.
+To prevent misconfiguration, Zeebe will check at startup that the specified bucket exists and can be accessed.
 
-[zstd]: https://github.com/facebook/zstd
-[commons-compress]: https://commons.apache.org/proper/commons-compress/
+Setting a `basePath` is not required but useful if you want to use the same bucket for multiple Zeebe clusters.
+When `basePath` is set, Zeebe will only create and access objects under this path.
+This can be any string that is a valid [object name](https://cloud.google.com/storage/docs/objects#naming), for example the name your cluster.
+
+Authentication is handled by [Application Default Credentials](https://cloud.google.com/docs/authentication/application-default-credentials).
```
[the remark this answers is not in the document]
Comment (oleschoenburg) [reply docs/self-managed/backup-restore/zeebe-backup-and-restore.md:106]: Yes, that's the plan at least AFAIK.

File: docs/self-managed/backup-restore/zeebe-backup-and-restore.md [hunk 2]
```diff
@@ -10,22 +10,25 @@ A backup of a Zeebe cluster is comprised of a consistent snapshot of all partiti
...
-### Configure S3 backup store
+This is especially relevant if you were using GCS through the S3 compatibility mode and want to switch to the new native support for GCS now.
+Even when the underlying storage bucket is the same, backups from one are not compatible with the other.
+:::
 
-Configure backup store in the configuration file as follows:
+### S3 backup store
 
-```
+To store your backups in any S3 compatible storage system such as [AWS S3] or [MinIO], set the backup store to `S3` and tell Zeebe how to connect to your bucket:
```
Comment (christinaausley) [inline docs/self-managed/backup-restore/zeebe-backup-and-restore.md:29]: Did you mean to link to AWS/MinIO directly here?

**Review by christinaausley — APPROVED** [review]
Just one non-blocking comment for clarification.

**Comment by christinaausley** [comment]
Just confirming these changes are only for the `next` version of docs?

File: docs/self-managed/backup-restore/zeebe-backup-and-restore.md (hunk 2 above)
Comment (oleschoenburg) [reply docs/self-managed/backup-restore/zeebe-backup-and-restore.md:29]: Yes, the links are here: https://github.com/camunda/camunda-platform-docs/pull/1882/files#diff-f1d372d47757830aae7a31653eae6b283c78a1a0d19719022886f6d7a1f47cf2R56-R57

**Comment by oleschoenburg** [comment]
> Just confirming these changes are only for the next version of docs?

Yes, the feature becomes available with 8.2.

**Comment by oleschoenburg** [comment]
Thanks both :pray: 

I'll let this merge automatically in the hope that I didn't break any more links and checks all turn green.

**Pull request merged**